cfn-resources/online-archive/docs/partitionfieldview.md

```diff
@@ -36,25 +34,13 @@ _Maximum Length_: <code>700</code>
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
-#### FieldType
-
-Data type of the parameter that that MongoDB Cloud uses to partition data. Partition parameters of type [UUID](http://bsonspec.org/spec.html) must be of binary subtype 4. MongoDB Cloud skips partition parameters of type UUID with subtype 3.
-
-_Required_: No
-
-_Type_: String
-
-_Allowed Values_: <code>date</code> | <code>int</code> | <code>long</code> | <code>objectId</code> | <code>string</code> | <code>uuid</code>
-
-_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
-
 #### Order
 
 Sequence in which MongoDB Cloud slices the collection data to create partitions. The resource expresses this sequence starting with zero. The value of the **criteria.dateField** parameter defaults as the first item in the partition sequence.
 
 _Required_: No
 
-_Type_: Integer
+_Type_: Double
```

Comment:
This is a breaking change. Do you have any reason for this change? This field seems to be an integer based on the example

Reply:
yes because this property was not being used, and in the mongo client, is a double not an integer, this is the client struct 
![image](https://github.com/mongodb/mongodbatlas-cloudformation-resources/assets/69170650/c62dedbd-372b-4733-926f-939ed3dc0a28)

Reply:
and there is no problem with the example, both the integer and double are represented the same way here is a test ive made:

here is the example file:
![image](https://github.com/mongodb/mongodbatlas-cloudformation-resources/assets/69170650/44ef6f37-4ba4-4c99-89c9-b785d9d45874)

and the result

<img width="1257" alt="image" src="https://github.com/mongodb/mongodbatlas-cloudformation-resources/assets/69170650/b701bb49-2803-472e-8c0c-b7a495b12dad">